core/slim_sim.cpp

```diff
@@ -4877,30 +4877,49 @@ void SLiMSim::WriteIndividualTable(table_collection_t *p_tables)
 {
     // This overwrites whatever might be previously in the individual table
     individual_table_clear(p_tables->individuals);
+    // First prepend the existing remembered individuals
+    individual_table_copy(tables.individuals, p_tables->individuals);
```

Comment:
oh, but I'm being dense. This function is called [here](https://github.com/MesserLab/SLiM/blob/master/core/slim_sim.cpp#L5291), on `output_tables`, which has *just* been copied from `tables`. So these two lines (`individual_table_clear` and `individual_table_copy`) are unnecessary. What we want to do is add some more lines to the end of the existing individual table.

Reply:
Perhaps this method should be called `AddCurrentGenerationToIndividuals` instead of `WriteIndividuals`.